libs/trainer/mlpe/trainer/trainer.py

```diff
@@ -191,15 +190,15 @@ def train(
         f["parameters"] = parameters.cpu().numpy()
 
     param_dim = parameters.shape[-1]
-    context_dim = strain.shape[-1] * strain.shape[-2]
+    _, n_ifos, strain_dim = strain.shape
```

Comment:
```suggestion
    param_dim, n_ifos, strain_dim = strain.shape
```

Reply:
No, so the parameters are separate. That dimension is the "batch" dimension (which we don't use)